src/DataMiner/SLManagedAutomation/UIBlockDefinition.cs

```diff
@@ -918,6 +918,26 @@ public string ValidationText
 		/// </example>
 		public bool WantsOnChange { get; set; }
 
+		/// <summary>
+		/// Gets or sets a value indicating whether changing the filter of this dialog box item will trigger a <see cref="UIResults.WasOnFilter"/> event.
+		/// </summary>
+		/// <value><c>true</c> if changing the filter of this dialog box item will trigger a <see cref="UIResults.WasOnFilter"/> event; otherwise, <c>false</c>.</value>
+		/// <example>
+		/// <remarks>
+		/// <para>Applicable only in case <see cref="Type"/> is set to <see cref="UIBlockType.DropDown"/>.</para>
+		/// <note type="note">Available from DataMiner 10.5.7/10.6.0 onwards, in Automation scripts launched from web apps and [specifying the `useNewIASInputComponents=true` URL parameter](xref:Configuring_app_actions_and_behavior_via_URL#configuring-app-behavior-via-the-url).</note> <!-- RN 42808 / RN 42845 -->
```

Comment:
This cross-reference does not get rendered correctly; apparently DocFX has trouble recognizing such cross-references within the automatically generated pages. We've recently encountered a similar issue where we could fix the issue by changing the cross-reference format (#4693). Could you check if you can adjust this similarly here? If not, it might be better to just use plain text and leave out the cross-reference.